src/scroll-timeline-base.js

```diff
@@ -512,18 +512,20 @@ export class ScrollTimeline {
 }
 
 function getScrollParent(node) {
-  if (node === undefined)
+  if (!node)
     return undefined;
 
-  if (node.scrollHeight > node.clientHeight ||
-      node.scrollWidth > node.clientWidth) {
-    return node;
-  } else {
-    return getScrollParent(node.parentNode);
-  }
+  const style = getComputedStyle(container);
+  let isScrollable = false;
+  style.overflow.split(" ").forEach(part => {
```

Comment:
You don't need to check both overflow directions. Per https://www.w3.org/TR/css-overflow-3/#propdef-overflow-y overflow-x and overflow-y will both compute to auto or hidden respectively if the other direction is not visible or clip, so you can just check style.overflowY and it will implicitly include non-visible non-clip overflow from X.

Reply:
done.